vizro-core/src/vizro/static/css/table.css

```
@@ -0,0 +1,32 @@
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  th {
+  background-color: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  td {
+  background-color: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  table {
+  --hover: var(--main-container-bg-color);
+}
+
+div.right_side
+  .dash-table-container
+  .dash-spreadsheet-container
+  .dash-spreadsheet-inner
+  td,
+.dash-table-container .dash-spreadsheet-container .dash-spreadsheet-inner th {
+  padding: 0px 4px;
+}
```

I just did a quick refinement, but it would be great if you can create a proper ticket to style the table properly as we're currently deviating from the initial design and I think the design eventually also has to be updated because the proposed spacing look quite large on the dashboard (I didn't implement the proposed spacing). 

```suggestion
div.right_side .dash-table-container .dash-spreadsheet-container .dash-spreadsheet-inner td .dash-cell-value.unfocused {
    color: var(--text-primary);
    font-size: 14px;
}

div.right_side .dash-table-container .dash-spreadsheet-container .dash-spreadsheet-inner .column-header-name {
    color: var(--text-primary); /* Should be text-secondary when hover color is implemented */
    font-size: 14px;
}

div.right_side .table-container {
 padding: 16px;
}
```